### ISSUE TITLE
`\ss` says a relation is missing when a table is empty and doesn't have any statistics

### ISSUE THREAD

**Issue by nineinchnick** [xo/usql#364]
```
pg:postgres@localhost:5432/benchto=> \ss benchmark_runs
Did not find any relation named "benchmark_runs".
pg:postgres@localhost:5432/benchto=> select count(*) from benchmark_runs;
┌───────┐
│ count │
├───────┤
│     0 │
└───────┘
```
It would be more valuable if it printed a summary row with zeros.

**Comment by nineinchnick** [comment]
How about a table with nulls? Having consistent output is important.

**Comment by kozikowskik** [comment]
Sorry, I think wasn't precise. I'm asking if it's not better to show a message instead of a summary with zeros if db hasn't returned any statistics for a table.

**Comment by nineinchnick** [comment]
When a database does not track statistics for all columns in a table, we should still show columns without statistics but with null values, not zeros. If there aren't any statistics for any column in a table, I'd still show this as a list of columns, because:
* it's more consistent - the user always sees the same output
* missing statistics is not an error and showing a message instead of _any_ results might suggest there's something wrong; statistics are usually just approximations and are collected in a best-effort manner, it's OK if they're missing
* you see a list of columns that you may recognize/expect and be sure you're checking the right table

**Comment by kozikowskik** [comment]
Ok, thank you for the explanation.

**Comment by nineinchnick** [comment]
Fixed in eea1b686d381b705f979f966a8f6d76eb3da3737
